Sources/Future/Containers/BidirectionalContainer.swift

```diff
@@ -13,4 +13,21 @@
 public protocol BidirectionalContainer<Element>: Container, ~Copyable, ~Escapable {
   func index(before i: Index) -> Index
   func formIndex(before i: inout Index)
+
+  @lifetime(borrow self)
+  func previousSpan(before index: inout Index) -> Span<Element>
+}
+
+@available(SwiftCompatibilitySpan 5.0, *)
+extension BidirectionalContainer where Self: ~Copyable & ~Escapable {
+  @lifetime(borrow self)
+  func previousSpan(before index: inout Index, maximumCount: Int) -> Span<Element> {
```

Comment:
This still has a `maximumCount` parameter?

Reply:
This is the definition of the utility helper that reintroduces it as a default extension. I forgot `@inlinable public` though 🙉